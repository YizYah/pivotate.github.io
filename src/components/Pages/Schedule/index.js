import React, { Component } from 'react';

class Schedule extends Component {
  componentDidMount() {
    const head = document.querySelector('head');
    const script = document.createElement('script');
    script.setAttribute(
      'src',
      'https://assets.calendly.com/assets/external/widget.js'
    );
    head.appendChild(script);
  }

  componentWillUnmount() {
    // whatever you need to cleanup the widgets code
  }

  render() {
    return (
      <div>
        <div className="schedule__img">
          <div className="schedule__content">
            <h2> We would like to meet you! </h2>
            <p>
              A friendly, no-commitment discussion of what you need in an app so
              we can help you get clarity, even if you don't build with us!
            </p>
          </div>
        </div>

        <div id="schedule_form">
          <div
            data-url="https://calendly.com/pivotate/clarify-new-app?hide_event_type_details=1&primary_color=16abbc
        "
            className="calendly-inline-widget"
            style={{ minWidth: '320px', height: '700px' }}
          />
        </div>
      </div>
    );
  }
}

export default Schedule;
