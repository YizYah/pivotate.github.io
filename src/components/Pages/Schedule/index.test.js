import React from 'react';
import renderer from 'react-test-renderer';
import Schedule from '../Schedule';

test('Snapshopt Test', () => {
  const component = renderer.create(<Schedule />);

  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
