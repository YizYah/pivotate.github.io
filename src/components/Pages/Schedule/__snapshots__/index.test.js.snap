// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshopt Test 1`] = `
<div>
  <div
    className="schedule__img"
  >
    <div
      className="schedule__content"
    >
      <h2>
         We would like to meet you! 
      </h2>
      <p>
        A friendly, no-commitment discussion of what you need in an app so we can help you get clarity, even if you don't build with us!
      </p>
    </div>
  </div>
  <div
    id="schedule_form"
  >
    <div
      className="calendly-inline-widget"
      data-url="https://calendly.com/pivotate/clarify-new-app?hide_event_type_details=1&primary_color=16abbc
 "
      style={
        Object {
          "height": "700px",
          "minWidth": "320px",
        }
      }
    />
  </div>
</div>
`;
